### PR TITLE
ADF-636: E2E - Improve file structure and usage

### PR DESCRIPTION
**Pull request by clagosarias**
Related to: https://oat-sa.atlassian.net/browse/ADF-636

- Move commands into different/new files depending on its scope.
- Delete unused code/selectors.

Related PR's
https://github.com/oat-sa/extension-tao-mediamanager/pull/324
https://github.com/oat-sa/extension-tao-itemqti/pull/1896
https://github.com/oat-sa/extension-tao-test/pull/394
https://github.com/oat-sa/extension-tao-item/pull/533

### CODE REVIEW

**Review by 404passionFruit — APPROVED** [review]
LGTM

File: views/cypress/support/manageSchema.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    cy.getSettled(classOptions).find('a[class="btn-info property-adder small"]').click();
+
+    cy.getSettled('span[class="icon-edit"]').last().click();
+    cy.get(propertyEdit).find('input').first().clear('input').type(newPropertyName);
+    cy.get(propertyEdit).find('select[class="property-type property"]').select('list');
+    cy.get(propertyEdit).find('select[class="property-listvalues property"]').select('Boolean');
+    cy.intercept('POST', `**/${editClassUrl}`).as('editClass');
+    cy.get('button[type="submit"]').click();
+    cy.wait('@editClass');
+});
```
Comment (habemuscode) [inline views/cypress/support/manageSchema.js:49]: end line

File: views/cypress/support/resourceProperties.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    selectOption,
+    treeRenderUrl) => {
+
+    cy.log('COMMAND: assignValueToProperty', nodeName, nodePropertiesForm);
+    cy.getSettled(`li [title ="${nodeName}"] a`).last().click();
+    cy.getSettled(nodePropertiesForm).find(selectOption).check();
+    cy.intercept('GET', `**/${treeRenderUrl}/getOntologyData**`).as('treeRender')
+    cy.getSettled('button[type="submit"]').click();
+    cy.wait('@treeRender');
+});
```
Comment (habemuscode) [inline views/cypress/support/resourceProperties.js:80]: End line

**Review by habemuscode — APPROVED** [review]
LGTM

**Pull request merged**